src/reputation/distributions.py

```diff
@@ -16,6 +23,90 @@ def amount(self):
     def gives_rep(self):
         return self._give_rep
 
+RSC_YEARLY_GIVEAWAY = 50000000
```

Comment:
I take it this will probably change year to year?

Reply:
yes, this amount will be variable. I think it's OK to set at the code level for now. We could also put it into an env var.

Reply:
Nah i think it is good here for us to see